client/js/components/statement/voter/StatementVoter.vue

```diff
@@ -378,6 +379,12 @@ export default {
       return isEmpty
     },
 
+    deleteEntry (index) {
```

Comment:
nitpick: I'd call this `handleDelete`, but maybe it's a matter of personal preference

Reply:
why not. e13ad3e588c43000eb5bbcbd90664bc9919e7c04